lib/midcom/helper/imagepopup/handler/upload.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * @author lukasz chalat
+ * @package midcom.helper.imagepopup
+ * @copyright The Midgard Project, http://www.midgard-project.org
+ * @license http://www.gnu.org/licenses/lgpl.html GNU Lesser General Public License
+ */
+
+/**
+ * This handler get uploaded file and save it in database and write to file.
+ *
+ * @package midcom.helper.imagepopup
+ */
+class midcom_helper_imagepopup_handler_upload extends midcom_baseclasses_components_handler
+{
+    /**
+     *
+     * @param mixed $handler_id
+     *            The ID of the handler.
+     * @param array $args
+     *            The argument list.
+     * @param
```

Comment:
can you put the phpdoc tags on one line each? Otherwise phpDocumentor won't recognize them afaik